cds_ils/cli.py

```diff
@@ -42,11 +50,17 @@
 
 CURRENT_DIR = pathlib.Path(__file__).parent.absolute()
 
+from cds_ils.literature.tasks import pick_identifier_with_cover_task
+
 
 @click.group()
 def fixtures():
     """Create initial data and demo records."""
 
+@click.group()
+def covers():
```

Comment:
Just a suggestion in the CLI commands:
`invenio covers recreate docid 123`
`invenio covers recreate docid --all`
so that you have the PID type and PID value, and you don't have to "guess" it via different commands

Reply:
fixed below in the command